### ISSUE TITLE
How to change the cross entropy loss function in "i3d head" to OHEM loss function?

### ISSUE THREAD

**Issue by Jmh0527** [open-mmlab/mmaction2#1614]
![image](https://user-images.githubusercontent.com/97495534/166137301-c774867a-c615-42f8-b66b-a4a80a6e0dce.png)

**Comment by kennymckormick** [comment]
Dear, Jmh0527,
Unfortunately you can not use the OHEMHingeLoss to train action recognition, basically it's designed for action localization. 
BTW, you can try to add some OHEM features to the CELoss, basically, it's about increasing the loss weights for hard examples.
